features/step_definitions/tag_steps.rb

```diff
@@ -124,6 +124,17 @@
   end
 end
 
+Given /^a period-containing tag "([^\"]*)" with(?: (\d+))? comments$/ do |tagname, n_comments|
+  tag = Fandom.find_or_create_by_name(tagname)
```

Comment:
Because the `tag` variable is apparently not used in the following code so it's not clear why that assignment is there at all?